package.json

```diff
@@ -3,11 +3,11 @@
   "scripts": {
     "lint": "eslint \"**/*.js\"",
     "format": "prettier \"**/*.{js,ts,tsx}\" --write --loglevel warn",
-    "test": "prettier \"**/*.{js,ts,tsx}\" --list-different && yarn lint && lerna run test"
+    "test": "prettier \"**/*.{js,ts,tsx}\" --list-different && yarn lint && ava"
```

Comment:
I know, it's not exactly the scope of this PR, but:

How about moving `yarn lint`, for instance, into a `posttest` script? Might make the scripts easier readable. Thoughts?

Reply:
{}

Reply:
```

Reply:
```

Reply:
```
{
  "test": "prettier \"**/*.{js,ts,tsx}\" --list-different && yarn lint && ava"
}
```
vs
```
{
  "pretest": "prettier \"**/*.{js,ts,tsx}\" --list-different && yarn lint",
  "test": "ava"
}
```

It barely improves anything 😄 If you want to make it shorter I can replace `--list-different` with `-l`.

Reply:
Ahhh, keep it 😅